src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -24,6 +24,26 @@ s32 func_80A87DC0(Actor* thisx, PlayState* play);
 void func_80A87FD0(EnTru* this, PlayState* play);
 void func_80A881E0(EnTru* this, PlayState* play);
 
+typedef enum {
+    /* 0x00 */ KOUME_ANIM_INJURED_LYING_DOWN1,
+    /* 0x01 */ KOUME_ANIM_INJURED_LYING_DOWN2,
+    /* 0x02 */ KOUME_ANIM_TRY_GET_UP,
+    /* 0x03 */ KOUME_ANIM_INJURED_RAISE_HEAD,
+    /* 0x04 */ KOUME_ANIM_INJURED_TALKING,
+    /* 0x05 */ KOUME_ANIM_INJURED_HEAD_UP1,
+    /* 0x06 */ KOUME_ANIM_INJURED_HEAD_UP2,
```

Comment:
Similar to above; is there any difference between these two?

Reply:
See https://github.com/zeldaret/mm/pull/1098#discussion_r985168795